src/Pages/3DPrintingConsole/requestForm.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './3D-console.css'
+import {
+  Button,
+  ButtonGroup,
+  Card,
+  CardTitle,
+  CardBody,
+  Collapse,
+  FormGroup,
+  Row,
+  Col
+} from 'reactstrap'
+
+function RequestForm (props) {
+  const jsonObject = props.item
```

Comment:
bro why are you calling it jsonObject
just do 
```suggestion
  const { item } = props
```
now `item` contains the stuff that props has

Reply:
Not fixed.